packages/atomic/src/components/commerce/atomic-commerce-pager/e2e/atomic-commerce-pager.e2e.ts

```diff
@@ -0,0 +1,144 @@
+import {test, expect} from './fixture';
+import {
+  assertAccessibility,
+  assertContainsComponentError,
+  assertDisabled,
+  assertEnabled,
+  assertPageInHash,
+  assertPagerSelected,
+} from './pager-assertions';
+
+test.describe('Default', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
+      'http://localhost:4400/iframe.html?id=atomic-commerce-pager--default&viewMode=story'
+    );
+  });
+
+  assertEnabled('nextButton');
```

Comment:
ehhh I dont particularly like this pattern we had for the old tests... I'd prefer just "normal" test directly in the file instead of those.

 What does everybody think ?

Reply:
We should adhere as much as possible to the practice of the test framework we're using as we do with code formatting with prettier.

Now for this specific issue/question:

My read from Playwright is that refactor and DRY come into three forms:
 - [Page Object](https://playwright.dev/docs/pom)
 - [Fixture ](https://playwright.dev/docs/test-fixtures)
 - [Test parametrization](https://playwright.dev/docs/test-parameterize) 
 
The first two work hand-in-hand for refactoring specific page aspects: they refactor **actions** and **locators** into smaller subsets as we do for our Headless controller. However, they do not refactor/encompass assertions.

The last one wraps and refactors the expected value used by assertions but not assertions.

So, in short, in my opinion
 - Locators should be refactored (more often than not)
 - Actionable may be refactored (if frequently used)
 - `test` and `expect` are barred from cross-file refactoring or being embedded in functions.
 
 > What I find frustrating is navigating through many functions just to get the assertion due to nested logic. However, if we keep assertions straightforward and flat within a separate file, I don't have an issue with it.
 
 Seeing `test` or `describe` as functions from an AST perspective is correct, but to interpret them as such as developers would be a mistake, I think.
 It's best to see a test suite as a novel:
 - It should be a readable piece (preferably pleasingly)
 - `describe` & `it` represents the headings that structure the novel.
 
 
 TL;DR:
 - Thee shall not embed assertions.
 - Thee shall not embed describe or test, with the exception of test parametrization _in a same file_ (think describe.each from Jest)